apps/src/code-studio/assets/SpriteUpload.js

```diff
@@ -1,41 +1,107 @@
 import React from 'react';
 
-export default class SpriteUpload extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      file: null
-    };
+const styles = {
```

Comment:
we're starting to standardize (while we're still using inline styles, at least) on moving `styles` to the bottom of the file